src/foam/lib/json/JSONParser.java

```diff
@@ -12,7 +12,7 @@
 public class JSONParser
   extends foam.core.ContextAwareSupport
 {
-  private Parser   parser   = new FObjectParser();
+  private Parser   parser   = new ExprParser();
```

Comment:
Updated. 

The `ExprParser` consists of an `Alt` with the two options being an `FObjectParser` and a custom parser for properties (`{ class: "__Property__". forClass_: "xyz" }`). This is needed to ensure properties are also parsed and handled correctly (the `__Property__` class isn't available from `foam.lookup()`, so the `FObjectParser` crashes if left as is).